tests/py/dynamo/models/test_export_serde.py

```diff
@@ -43,9 +43,9 @@ def forward(self, x):
 
     exp_program = torchtrt.dynamo.trace(model, **compile_spec)
     trt_module = torchtrt.dynamo.compile(exp_program, **compile_spec)
-    torchtrt.save(trt_module, "/tmp/trt.ep", inputs=[input])
+    torchtrt.save(trt_module, "./trt.ep", inputs=[input])
```

Comment:
Can we either use the tempdir or .gitignore this file

Reply:
Fixed